src/vs/base/browser/dom.ts

```diff
@@ -944,7 +944,10 @@ export function getActiveWindow(): CodeWindow {
 
 export function getFocusedWindow(): CodeWindow | undefined {
 	const document = getActiveDocument();
-	return (document.defaultView?.window) as CodeWindow | undefined;
+	if (document.defaultView?.window.document.hasFocus()) {
```

Comment:
I think this could benefit from a comment why this change is needed from what we had before?

Reply:
@meganrogge that is not a comment at all that helps understand what you are doing

Reply:
@bpasero I feel the code speaks for itself here - esp with the function title. What am I missing?

Reply:
Lets continue in https://github.com/microsoft/vscode/issues/249416 that I created based on that.